api/Navigator.json

```diff
@@ -1717,25 +1719,28 @@
               "notes": "Always returns <code>undefined</code>."
             },
             "opera": {
-              "version_added": true
+              "version_added": "15",
+              "notes": "Always returns <code>20030107</code>."
             },
             "opera_android": {
-              "version_added": true
+              "version_added": "14",
+              "notes": "Always returns <code>20030107</code>."
             },
             "safari": {
-              "version_added": true,
+              "version_added": "1",
```

Comment:
Ah, so it turns out "20030107" is about 5 months before Safari 1 was released. Pretty sure that's where the date came from.